frontend/pages/policies/PolicyPage/PolicyPage.tsx

```diff
@@ -151,15 +140,14 @@ const PolicyPage = ({
     }
   );
 
-  useEffect(() => {
-    const detectIsFleetQueryRunnable = () => {
-      Fleet.status.live_query().catch(() => {
-        setIsLiveQueryRunnable(false);
-      });
-    };
```

Comment:
Not clear on why the function was being defined inside of the `useEffect`. Does not appear to be necessary.